data/pkg.go

```diff
@@ -318,8 +319,10 @@ const (
 	OPT_PAR_MSG_PAYLOAD_MAX = 1500
 
 	// User Data Header
-	UDH_CONCAT_MSG_8_BIT_REF  byte = 0x00
-	UDH_CONCAT_MSG_16_BIT_REF byte = 0x08
+	UDH_CONCAT_MSG_8_BIT_REF    byte = 0x00
+	UDH_CONCAT_MSG_8_BIT_REF_2  byte = 0x05
+	UDH_CONCAT_MSG_16_BIT_REF_2 byte = 0x06
```

Comment:
Hi @septs can I ask where did you find the reference for these bytecodes ?

Reply:
see https://en.wikipedia.org/wiki/User_Data_Header#UDH_Information_Elements